component/answer/src/main/java/org/exoplatform/management/answer/operations/AnswerDataImportResource.java

```diff
@@ -113,62 +113,69 @@ public void execute(OperationContext operationContext, ResultHandler resultHandl
   @SuppressWarnings("unchecked")
   private void importAnswerData(File file, File spaceMetadataFile, boolean replaceExisting, boolean createSpace) throws Exception {
     List<Object> objects = (List<Object>) deserializeObject(file, null, null);
-    Category category = (Category) objects.get(0);
-    String parentId = category.getPath().replace("/" + category.getId(), "");
-    List<Question> questions = (List<Question>) objects.get(1);
-    Category parentCategory = faqService.getCategoryById(parentId);
-    if (parentCategory == null) {
-      log.warn("Parent Answer Category of Category '" + category.getName() + "' doesn't exist, ignore import operation for this category.");
-      return;
-    }
+    for(int i=0 ; i<objects.size(); i+=2) {
```

Comment:
for(int i=0 ; i<objects.size(); i+=2) {
      Category category = (Category) **objects.get(i)**;
      String parentId = category.getPath().replace("/" + category.getId(), "");
      List<Question> questions = (List<Question>) **objects.get(i+1)**;

The Object i is the category 
The Object i+1 is the list of category questions